src/math/Color.js

```diff
@@ -601,9 +603,9 @@ class Color {
 }
 
 Color.NAMES = _colorKeywords;
-Color.prototype.isColor = true;
 Color.prototype.r = 1;
 Color.prototype.g = 1;
 Color.prototype.b = 1;
```

Comment:
What about these? Changing them, too?
```js
Color.prototype.r = 1;
Color.prototype.g = 1;
Color.prototype.b = 1;
```

Reply:
I was gonna, but since these properties are writable, I tought it made sense to set them as class fields e.g.

```js
class Color {

  r = 1;
  g = 1;
  b = 1;

  constructor( r, g, b ) {
    // ...
```

However we can't do that yet, see https://github.com/mrdoob/three.js/issues/19986#issuecomment-675559937